ts/src/pro/binance.ts

```diff
@@ -4307,7 +4307,8 @@ export default class binance extends binanceRest {
             client.reject (message, id);
         }
         // reset connection if 5xx error
-        if (this.safeString (code, 0) === '5') {
```

Comment:
this was an incorrect approach, as "code" might be treated like as object/array across some langs, and `0` would not locate first char of the string, but first item in array